go-controller/pkg/config/cni.go

```diff
@@ -18,14 +18,13 @@ type OVNNetConf struct {
 }
 
 // WriteCNIConfig writes a CNI JSON config file to directory given by global config
-func WriteCNIConfig(configFilePath string) error {
+func WriteCNIConfig() error {
 	bytes, err := json.Marshal(&OVNNetConf{
 		NetConf: types.NetConf{
 			CNIVersion: "0.3.1",
 			Name:       "ovn-kubernetes",
 			Type:       CNI.Plugin,
 		},
-		ConfigFilePath: configFilePath,
```

Comment:
You can remove the ConfigFilePath item from OVNNetConf too now.  But that means there's nothing interesting there, so you can just remove OVNNetConf entirely and switch to "types.NetConf" from the CNI package.